fetch.js

```diff
@@ -45,40 +53,41 @@ var data = JSON.stringify({
     }
 }
 `
-});
-const default_options = {
-  hostname: "api.github.com",
-  path: "/graphql",
-  port: 443,
-  method: "POST",
-  headers: {
-    Authorization: `Bearer ${GITHUB_TOKEN}`,
-    "User-Agent": "Node"
-  }
-};
+  });
+  const default_options = {
+    hostname: "api.github.com",
+    path: "/graphql",
+    port: 443,
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${GITHUB_TOKEN}`,
+      "User-Agent": "Node"
+    }
+  };
 
-const req = https.request(default_options, res => {
-  let data = "";
+  const req = https.request(default_options, res => {
+    let data = "";
 
-  console.log(`statusCode: ${res.statusCode}`);
-  if (res.statusCode != 200) {
-    throw "The request to Github didn't suceed. Maybe check Github Token?";
-  }
+    console.log(`statusCode: ${res.statusCode}`);
+    if (res.statusCode !== 200) {
+      throw new Error(ERR.requestFailed);
+    }
```

Comment:
is this a typo? or an operator which exists in JS and I didn't know about?
```suggestion
    if (res.statusCode != 200) {
      throw new Error(ERR.requestFailed);
    }
```

Reply:
Not a typo, it's for not equal value or not equal type. Think of it as opposite of the '===' operator.

Reply:
looks like my knowledge about JS isn't that great